pootle/apps/pootle_project/models.py

```diff
@@ -301,6 +301,11 @@ def config(self):
         return ObjectConfig(self)
 
     @property
+    def local_fs_path(self):
+        return os.path.join(
+            settings.POOTLE_FS_PATH, self.code)
```

Comment:
@phlax I've just realized that we don't have any description for `settings.POOTLE_FS_PATH`